packages/lib/src/components/PayPal/Paypal.tsx

```diff
@@ -57,7 +57,7 @@ class PaypalElement extends UIElement<PayPalElementProps> {
     }
 
     updateWithAction(action: PaymentAction) {
-        if (action.paymentMethodType !== this.data.paymentMethod.type) throw new Error('Invalid Action');
+        if (action.paymentMethodType !== this.type) throw new Error('Invalid Action');
```

Comment:
Was this line always incorrect in the old code?

Reply:
`this.data.paymentMethod.type` is the same as `this.type` . 

I had to update this because Typescript started complaining after adding the return type in the `get data()` method